qemu/tests/iozone_linux.py

```diff
@@ -0,0 +1,28 @@
+from virttest import error_context
+from provider.storage_benchmark import generate_instance
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run IOzone for linux on a linux guest:
+    1) Log into a guest.
+    2) Execute the IOzone test.
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    iozone_test_dir = params.get('iozone_test_dir', '/home')
```

Comment:
`iozone_test_dir = params.get('iozone_test_dir', '/home')`
`iozone_test_dir = '/home'`
It is kind of redundant, better make sure it is optional or not.

Reply:
thanks, have updated.